lib/components/form/batch-styled.ts

```diff
@@ -94,13 +106,19 @@ export const BatchPreferencesContainer = styled.div`
   margin-top: 50px;
   padding: 5px 10px;
 `
-
 export const MainSettingsRow = styled.div`
   align-items: top;
   display: flex;
   flex-direction: row;
-  justify-content: flex-start;
+  justify-content: space-between;
   margin-bottom: 5px;
+
+  @media (max-width: 355px), (min-width: 991px) and (max-width: 1110px) {
+    display: grid;
```

Comment:
This produces the behavior in the screenshot below. Is that desired?

<img width="348" alt="image" src="https://user-images.githubusercontent.com/56846598/232602580-b2a46c52-eb0f-4397-845e-83ddd372fb22.png">